plugins/org.jkiss.dbeaver.ext.altibase/OSGI-INF/l10n/bundle.properties

```diff
@@ -273,8 +273,18 @@ tree.constraints.node.tip          = Constraints
 tree.dataType.node.name            = Data Type
 tree.dataTypes.node.name           = Data Types
 tree.dataTypes.node.tip            = Global data types
+<<<<<<< HEAD
```

Comment:
Please fix broken merge

Reply:
Thanks for pointing out the problem :) I merge and commit the file.

Anyway, my Eclipse doesn't show any conflict at bundle.properties. If you know how to do that, please let me know in order to prevent broken merge :)

Reply:
Hello @ShadelessFox, 

I resolved the broken issue. Could you check it out? Thanks :)